.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,56 @@
+name: Static Analysis
```

Comment:
Let's just add the job to the `coding-standards` workflow, that was just merged.